data/bigquery_sharded_data_pull.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
can you change the base to rel-1.8.5? I would hope that we push real fix for 1.8.6

Reply:
Base should be changed.